tools/backups2datalad/datasetter.py

```diff
@@ -174,7 +173,7 @@ async def sync_dataset(
         ds: AsyncDataset,
         manager: Manager,
         error_on_change: bool = False,
-    ) -> tuple[bool, dict[str, DatasetStats]]:
+    ) -> bool:
```

Comment:
Also adjust the "Returns" comment below.

Reply:
thanks, done - removed that 2nd line